redis/redis.go

```diff
@@ -232,8 +232,8 @@ func (p *RedisProvider) Read(sid string) (session.RawStore, error) {
 
 // Exist returns true if session with given ID exists.
 func (p *RedisProvider) Exist(sid string) bool {
-	has, err := p.c.Exists(p.prefix + sid).Result()
-	return err == nil && has
+	v, err := p.c.Exists(p.prefix + sid).Result()
+	return err == nil && v == 1
```

Comment:
What does this `v` hold? The number of results found? If so, shouldn't we check `v >= 1` to be resilient?

Reply:
@unknwon renamed it to "count" - as per definiton of the Result function for IntCmd (go-redis) ... this can only be 0 or 1 with one argument